src/main/resources/application.yml

```
@@ -8,6 +8,7 @@ spring:
     open-in-view: false
     properties:
       hibernate:
+        dialect: org.hibernate.dialect.MySQLDialect
         format_sql: true
         dialect: org.hibernate.dialect.MySQLDialect
```

YAMLlint (1.38.0)

[error] duplication of key "dialect" in mapping

(key-duplicates)